projects/levvy-fi-tokens/index.js

```diff
@@ -0,0 +1,36 @@
+const { post } = require("../helper/http");
+
+module.exports = {
+  methodology:
+    "Counts ADA locked in token lending offer pools, and tokens locked as collateral in loans",
+  misrepresentedTokens: true,
+  cardano: {
+    tvl: async () => {
+      const data = await post(
+        "https://citizens.theapesociety.io/api/getLevvyData",
+        {}
+      );
+      return {
+        cardano: data.tokens.tvl,
```

Comment:
what is included in the tvl here? does not match the vaule on your site: 
![image](https://github.com/DefiLlama/DefiLlama-Adapters/assets/99907941/b8c5755b-a00e-42c6-bf60-964fb521af04)
Also, is tvl: total supplied or total supplied - total borrowed?

Reply:
One the site, that number doesn't currently include the collateral locked in active loans.  TVL in this commit equals total in pools + total value of collateral in active loans